docs/audit-log-filter-variables.md

```diff
@@ -0,0 +1,745 @@
+    +----------------------------------------------------+
+    | {"timestamp" : "2023-06-02 09:43:25", "id": 10 }   |
+    +----------------------------------------------------+
+    ```
+
+### audit_log_filter_remove_filter(filter_name)
+
+This function removes the selected filter from the current set of filters.
+
+If user accounts are assigned the selected filter, the user accounts are no longer filtered. The user accounts are removed from `audit_log_filter_user`. If the user accounts are in a current session, they are detached from the selected filter and no longer logged.
+
+#### Parameters
+
+`filter_name` - a selected filter name as a string.
+
+
+#### Returns
+
+This function returns either an `OK` for success or an error message for failure.
+
+If the filter name does not exist, no error is generated. 
+
+
+#### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT audit_log_filter_remove_filter('filter-name');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +------------------------------------------------+
+    | audit_log_filter_remove_filter('filter-name')  |
+    +------------------------------------------------+
+    | OK                                             |
+    +------------------------------------------------+
+    ```
+
+### audit_log_filter_remove_user(user_name)
+
+This function removes the assignment of a filter from the selected user account.
+
+If the user account is in a current session, they are not affected. New sessions for this user account use the default account filter or are not logged.
+
+If the user-name is `%`, the default account filter is removed.
+
+#### Parameters
+
+`user_name` - a selected user name in either the `user_name`@`host_name` format or `%`.
+
+
+#### Returns
+
+This function returns either an `OK` for success or an error message for failure.
+
+If the user_name has no filter assigned, no error is generated. 
+
+
+#### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT audit_log_filter_remove_user('user-name@localhost');
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    +------------------------------------------------------+
+    | audit_log_filter_remove_user('user-name@localhost')  |
+    +------------------------------------------------------+
+    | OK                                                   |
+    +------------------------------------------------------+
+    ```
+
+### audit_log_rotate()
+
+#### Parameters
+
+None.
+
+#### Returns
+
+This function returns the new file name.
+
+#### Example
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT audit_log_filter_rotate();
+```
+
+
+### audit_log_filter_set_filter(filter_name, definition)
+
+This function, when provided with a filter name and definition, adds the filter. 
+
+The new filter has a different filter ID. If the filter name exists and is active within any current sessions, those sessions are detached and no longer logged.
```

Comment:
Please remove this. In our case there will be an error if filter with the same name already exists.